swagger.json

```diff
@@ -3369,6 +3369,17 @@
               "items": {
                 "type": "object",
                 "properties": {
+                  "id": {
+                    "type": "integer",
+                      "format": "int64",
```

Comment:
Nit: formatting looks slightly off.

Reply:
🤦‍♂️ I think my vscode settings for this project are messed up, thanks for calling that out!

Reply:
fixed!